renovate.json

```diff
@@ -37,7 +37,7 @@
   },
   {
     "matchPackageNames": ["org.asciidoctor.jvm.convert", "org.asciidoctor.jvm.pdf"],
-    "allowedVersions": "<=3.3.2"
+    "allowedVersions": "<=4.0.4"
   },
```

Comment:
how about we just remove this? An auto-upgrade from 4.0.4 to 4.0.5 would have worked, for example